Nodejs/Product/Nodejs/Project/MigrateToJspsInfoBar.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Imaging;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudioTools;
+
+namespace Microsoft.NodejsTools.Project
+{
+    internal static class MigrateToJspsInfoBar
+    {
+        private static IVsInfoBarUIElement CurrentInfoBarElement;
+
+        public static void Show(IVsShell vsShell, IVsInfoBarUIFactory infoBarUiFactory)
+        {
+            if (ErrorHandler.Failed(vsShell.GetProperty((int)__VSSPROPID7.VSSPROPID_MainWindowInfoBarHost, out var tmp)))
+            {
+                return;
+            }
+            var infoBarHost = (IVsInfoBarHost)tmp;
+
+            CurrentInfoBarElement?.Close();
+
+            var infoBarModel = new InfoBarModel(Resources.MigrateToJspsPrompt);
```

Comment:
It would be a nice user experience if there was a button on the gold bar that users can click that would do the migration instead of giving them the steps to the project context menu. I think that should be possible by just passing in an `Action` into this ctor that does it.

Reply:
Also moving this to next iteration, and merging for now.

Reply:
Added in #2493 !